static/css/restyle/restyle.less

```diff
@@ -1823,10 +1822,25 @@ h3.author .transfer-ownership {
   }
 }
 
-.statistics .primary {
-  float: left;
-  margin-left: 0;
-  width: 77%;
+.restyle.statistics {
+  #page {
+    max-width: @pageWidth;
+    min-width: @pageWidth;
```

Comment:
Overriding. The old CSS sets a max-width of 1280px on this one page, but it makes it look weird. This brings it in line with the rest f the site.